tests/test_json_schema.py

```diff
@@ -1292,6 +1308,26 @@ class MyGenerator(GenerateJsonSchema):
     assert model_schema['properties'] == properties
 
 
+def test_byte_size_type():
+    class Model(BaseModel):
+        a: ByteSize
+
+    assert Model.model_json_schema() == {
```

Comment:
Test looks great. Could you please add a test verifying that `Model.model_json_schema(mode='serialization')` has the results that we expect, as well? Thanks!

Reply:
forgot to commit that initially. added a test and function for serialization. however, the output is exactly the same for validation/serialization. is that expected behavior?